egnoramoose.py

```diff
@@ -1,119 +1,167 @@
 class Pawn:
-	def __init__(self,location):
-		self.location = location
+    pass
+
+
+class Location:
+    def __init__(self, *location):
+        self.row = -1
+        self.column = -1
+        self._convert(location)
+
+    def __sub__(self, other):
+        return Location(self.row-other.row, self.column-other.column)
+
+    def __str__(self):
```

Comment:
This allows us to do str(location_object) and display something readable. As you see, I made it the same format we want it to be displayed on the board